apple/Sources/FerrostarSwiftUI/Views/Maneuver/DefaultIconographyManeuverInstructionView.swift

```diff
@@ -54,6 +54,8 @@ public struct DefaultIconographyManeuverInstructionView: View {
                     maneuverModifier: maneuverModifier
                 )
                 .frame(maxWidth: 48)
+                // REVIEW: without this, the first image in the vstack was rendering very small. Curiously subsequent items in the vstack looked reasonable.
```

Comment:
At the end of the day I don't think it's dangerous to include this change, but it's curious.

Here's what it looks like with this change reverted:

https://github.com/user-attachments/assets/cf2b9280-fe2e-4971-803f-621e3b8b4d4d